mypy.ini

```diff
@@ -9,26 +10,29 @@ exclude = (?x)(
     ^e2e_playwright/compilation_error_dialog\.py$
   )
 
-disallow_incomplete_defs = true
+# strict mode settings:
+warn_unused_configs = true
+disallow_any_generics = true
 disallow_subclassing_any = true
-disallow_untyped_decorators = true
 disallow_untyped_calls = true
 disallow_untyped_defs = true
-allow_redefinition = true
+disallow_incomplete_defs = true
+disallow_untyped_decorators = true
 check_untyped_defs = true
-disallow_any_generics = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+warn_return_any = true
 no_implicit_reexport = true
-no_implicit_optional = true
-scripts_are_modules = true
 strict_equality = true
+extra_checks = true
+
+# other settings:
+allow_redefinition = false
+no_implicit_optional = true
 strict_optional = true
-warn_redundant_casts = true
-warn_return_any = true
-warn_unused_configs = true
-warn_unused_ignores = true
 warn_unreachable = true
 warn_no_return = true
-extra_checks = true
```

Comment:
Mostly just some reshuffling to make it clear which settings are part of the `strict` settings for mypy